src/esphome/sensor/cse7766.cpp

```diff
@@ -15,6 +15,7 @@ void CSE7766Component::loop() {
   const uint32_t now = millis();
   if (now - this->last_transmission_ >= 500) {
     // last transmission too long ago. Reset RX index.
+    ESP_LOGV(TAG, "%d milliseconds have passed since the last transmission.  Resetting index", now);
```

Comment:
```suggestion
    ESP_LOGV(TAG, "%u milliseconds have passed since the last transmission.  Resetting index", now - this->last_transmission_);
```

Technically the log statement is incorrect. It can also be dropped though.